build.sbt

```diff
@@ -87,4 +81,7 @@ developers := List(
   )
 )
 
-enablePlugins(GitVersioning)
+publishTo := Some(Resolver.evolutionReleases)
+
+//addCommandAlias("check", "all versionPolicyCheck Compile/doc")
+addCommandAlias("check", "show version")
```

Comment:
is this needed?

Reply:
`check` is called by action - for now, here we define it as "no-op", in future we can provide full set of checks